website/src/queries-init.js

```diff
@@ -21,7 +22,6 @@ const UF_MSG_API_RATE     = "<b>GitHub API rate-limits exceeded.</b> Consider pr
 // list of messages which should not be cleared when the request ends
 const UF_PRESERVED_MSGS = [
     UF_MSG_NO_FORKS,
-    UF_MSG_EMPTY_FILTER,
```

Comment:
Why did you need to take this one out?

Reply:
Why did you think it "error"?

```js
// list of messages which should not be cleared when the request ends
const UF_PRESERVED_MSGS = [
  UF_MSG_NO_FORKS,
  UF_MSG_ERROR,
  UF_MSG_API_RATE
];
```

```js
function clearNonErrorMsg() {
  const msg = JQ_ID_MSG.html();
  if (!UF_PRESERVED_MSGS.includes(msg))
    clearMsg();
}
```

Firstly let's ignore the gap between `error` and `preserved`.

When the filter updates, certainly we need to clear some outdated msgs and only preserve. And the msg `empty` could be outdated because the table may switch between empty and non-empty as I edit the filter field.

Reply:
Ah, I must've forgotten to adapt the method's name at some point.
You are right that it shouldn't mention "error".